src/main/kotlin/net/maizegenetics/phgv2/pathing/DiploidEmissionProbability.kt

```diff
@@ -88,18 +88,24 @@ class DiploidEmissionProbability(val readMap: Map<ReferenceRange, Map<List<Strin
     /**
      * Calculates the emission probability for a pair of haplotypes. The emission probability is the probability of
      * observing this particular set of read counts given that they were generated by this haplotype pair.
+     * UnorderedHaplotypePair is used because the order of the haplotypes does not matter.
      */
     private fun haplotypePairProbability(haplotypes: UnorderedHaplotypePair, readCounts: Map<Set<String>, Int>): Double {
-
+        //TODO   add comments explaining calculations
```

Comment:
Did you want to do this?

Reply:
Yes. I am adding an explanation of DiploidEmissionProbability calculation.